docs/contributing.rst

```diff
@@ -274,13 +274,14 @@ enabling and disabling the instrumentation and overriding the service name.
 Library support
 ~~~~~~~~~~~~~~~
 
-``ddtrace`` tries to support as many active versions of a library as possible.
-The general rule is:
+The ``ddtrace`` library's support policy for third-party libraries we provide integrations for is as follows:
 
-  - If the integration depends on internals of the library then test every
-    minor version going back 2 years.
 
-  - Else test each major version going back 2 years.
+  - Test the oldest and latest minor versions of the most latest major version going back 2 years.
+
+  - Test the latest minor version of any previous major version going back 2 years.
```

Comment:
I wonder if we should re-word as "what we support" vs "what we test".

Since supporting every version going back 2 years vs how we test that can be different.

e.g. we aren't support only the oldest and latest minor versions, we are supporting everything in between, but we are only testing those versions.


I don't have better wording, so take this as just an optional nit for now.